include/RE/H/HUDNotifications.h

```diff
@@ -12,29 +15,24 @@ namespace RE
 	public:
 		inline static constexpr auto RTTI = RTTI_HUDNotifications;
 
-		struct UnkData1
+		class Notification
 		{
-			struct UnkData2
-			{
-				void*         unk00;  // 00
-				std::uint64_t unk08;  // 08
-				void*         unk10;  // 10
-				std::uint64_t unk18;  // 18
-			};
-			static_assert(sizeof(UnkData2) == 0x20);
+		public:
+			Notification() :
+				type(0), quest(nullptr), word(nullptr), time(0) {}
+			Notification(const Notification& a_rhs);
```

Comment:
There's not need to define a copy ctor here, the compiler generated one is perfectly fine.